<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>778</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IGI 3DDigitizer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="IGI3DDigitizer.qrc">
    <normaloff>:/Images/Images/icon.png</normaloff>:/Images/Images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(0,qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255)) 0, 0);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item>
     <widget class="QWidget" name="mainWidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QGroupBox" name="Setup">
           <property name="title">
            <string>Setup</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="CameraCalibFileButton">
              <property name="text">
               <string>Select Camera Calibration File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="CameraCalibFileEdit">
              <property name="text">
               <string>&lt;...&gt;</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PointerCalibFileButton">
              <property name="text">
               <string>Select Pointer Tool Calibration File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="PointerCalibFileEdit">
              <property name="text">
               <string>&lt;...&gt;</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="layoutHozPointer">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Select pointer ID:     </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="PointerIDBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>105</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>90</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="iconSize">
                 <size>
                  <width>40</width>
                  <height>40</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>403</string>
                 </property>
                 <property name="icon">
                  <iconset resource="IGI3DDigitizer.qrc">
                   <normaloff>:/Images/Images/403.png</normaloff>:/Images/Images/403.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>102</string>
                 </property>
                 <property name="icon">
                  <iconset resource="IGI3DDigitizer.qrc">
                   <normaloff>:/Images/Images/102.png</normaloff>:/Images/Images/102.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>201</string>
                 </property>
                 <property name="icon">
                  <iconset resource="IGI3DDigitizer.qrc">
                   <normaloff>:/Images/Images/201.png</normaloff>:/Images/Images/201.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>202</string>
                 </property>
                 <property name="icon">
                  <iconset resource="IGI3DDigitizer.qrc">
                   <normaloff>:/Images/Images/202.png</normaloff>:/Images/Images/202.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>502</string>
                 </property>
                 <property name="icon">
                  <iconset resource="IGI3DDigitizer.qrc">
                   <normaloff>:/Images/Images/502.png</normaloff>:/Images/Images/502.png</iconset>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="LEDlabelTool">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="IGI3DDigitizer.qrc">:/Images/Images/redLED.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="layoutHozReference">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Select reference ID:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="RefIDBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>105</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>90</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="iconSize">
                 <size>
                  <width>40</width>
                  <height>40</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="icon">
                  <iconset resource="IGI3DDigitizer.qrc">
                   <normaloff>:/Images/Images/noImg.png</normaloff>:/Images/Images/noImg.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>102</string>
                 </property>
                 <property name="icon">
                  <iconset resource="IGI3DDigitizer.qrc">
                   <normaloff>:/Images/Images/102.png</normaloff>:/Images/Images/102.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>403</string>
                 </property>
                 <property name="icon">
                  <iconset resource="IGI3DDigitizer.qrc">
                   <normaloff>:/Images/Images/403.png</normaloff>:/Images/Images/403.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>201</string>
                 </property>
                 <property name="icon">
                  <iconset resource="IGI3DDigitizer.qrc">
                   <normaloff>:/Images/Images/201.png</normaloff>:/Images/Images/201.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>202</string>
                 </property>
                 <property name="icon">
                  <iconset resource="IGI3DDigitizer.qrc">
                   <normaloff>:/Images/Images/202.png</normaloff>:/Images/Images/202.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>502</string>
                 </property>
                 <property name="icon">
                  <iconset resource="IGI3DDigitizer.qrc">
                   <normaloff>:/Images/Images/502.png</normaloff>:/Images/Images/502.png</iconset>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="LEDlabelDRF">
                <property name="maximumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="IGI3DDigitizer.qrc">:/Images/Images/redLED.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="InitializeButton">
           <property name="text">
            <string>Initialize</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CheckPositionButton">
           <property name="text">
            <string>Check Camera View</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="PointAquisition">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
             <property name="title">
              <string>Point Aquisition</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="StartButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>Start</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ClearButton">
                <property name="text">
                 <string>Clear</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="InfoLayerHoz">
                <item>
                 <widget class="QLabel" name="PointNr">
                  <property name="text">
                   <string>Point No. </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="editPointNr">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Displayed:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="editVisNr">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="layoutLoadSave">
                <item>
                 <widget class="QPushButton" name="LoadButton">
                  <property name="text">
                   <string>Load</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="SaveAsButton">
                  <property name="text">
                   <string>Save As...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="QuitPushButton">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="IGI3DDigitizer.qrc">
                 <normaloff>:/Images/Images/ButtonQuit.png</normaloff>:/Images/Images/ButtonQuit.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <property name="shortcut">
                <string>Q</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QLabel" name="logoLabel">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="IGI3DDigitizer.qrc">:/Images/Images/SZI_V_cmyk.png</pixmap>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="helpButton">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="IGI3DDigitizer.qrc">
                 <normaloff>:/Images/Images/help.png</normaloff>:/Images/Images/help.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="shortcut">
                <string>H</string>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetView" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="igstk::QTWidget" name="Display3D" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/Images/Images/splash.bmp);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>igstk::QTWidget</class>
   <extends></extends>
   <header>igstkQTWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="IGI3DDigitizer.qrc"/>
 </resources>
 <connections/>
</ui>
